spring-cloud-dataflow-platform-kubernetes/src/test/java/org/springframework/cloud/dataflow/server/config/kubernetes/KubernetesTaskPlatformFactoryTests.java

```diff
@@ -62,6 +62,36 @@ public void kubernetesTaskPlatformNoScheduler() {
 
 	@Test
 	public void kubernetesTaskPlatformWithScheduler() {
+		KubernetesPlatformProperties platformProperties = new KubernetesPlatformProperties();
+		KubernetesDeployerProperties deployerProperties = new KubernetesDeployerProperties();
+		deployerProperties.getLimits().setMemory("5555Mi");
+		KubernetesTaskLauncherProperties taskLauncherProperties = new KubernetesTaskLauncherProperties();
+		taskLauncherProperties.setBackoffLimit(5);
```

Comment:
Where do we test for the backofflimit?

Reply:
Good catch. We aren't testing backoffLimit in this test case rather the next one does. I kept this test as it used to be (except adding the new properties arg)